pkg/controllers/federate/util.go

```diff
@@ -203,6 +203,7 @@ var (
 		common.FollowersAnnotation,
 	)
 
+	// TODO: Do we need to specify the internal annotations here?
```

Comment:
are these the annotations that will be ignored when generating the template for the federated object?

if that is the case and we do not specify the internal annotations here, would it result in unnecessary updates to the federated object, and consequently unnecessary syncs to the member clusters when the internal annotations change?

Reply:
No, as discussed offline, these are the annotations that should be ignored when computing template annotations from **source annotations**. For annotations that only appear on the federated object, there's no need to include them here. In fact the internal annotations listed here are redundant since they only appear on the federated object.